### ISSUE TITLE
Mismatch between IRC URL and default Hexchat config

### ISSUE THREAD

**Issue by rmunn** [linuxmint/mintwelcome#19]
The welcome screen's "Chat room" button, by default, goes to `irc://irc.spotchat.org/linuxmint-help`. I have set up HexChat to use SASL to connect to the SpotChat network with my NickServ password. However, HexChat appears to use the hostname **and** port number to identify the network, because when I connect via that URL, HexChat does not use SASL and does not send my NickServ password.

Looking into it, I found that there is no entry in the `.config/hexchat/servlist.conf` for plain "irc.spotchat.org" without any port number. If I edit the "Linux Mint" server configutation at the top of the `servlist.conf` file and add a `S=irc.spotchat.org` line, then HexChat will use my SASL configuration and send my password.

Two possible solutions to this mismatch:

1) Edit the Welcome screen IRC URL to include the port number, e.g. `irc://irc.spotchat.org:6667/linuxmint-help`. In theory this should work -- but in practice, I've tried this and it doesn't seem to make HexChat recognize that I've configured that network to use SASL.

2) Add the line `S=irc.spotchat.org` to the Linux Mint network configuration at the top of `/etc/skel/.config/hexchat/servlist.conf`. I've tried this and it DOES work: when I have that line in my `/home/rmunn/.config/hexchat/servlist.conf` file, HexChat figures out that `irc://irc.spotchat.org/linuxmint-help` is trying to open the server that I have configured a password for, and identifies me via SASL when I connect to the server.

Solution 1 would involve a change to `mintwelcome.py`. Solution 2 would involve a change to `/etc/skel/.config/hexchat/servlist.conf`. I know that that file is part of the `mint-common-artwork` Debian package, but I've been unable (so far) to find which Git repository tracks the contents of `mint-common-artwork`. So even though this repo isn't _quite_ the right place to report this bug, I'm reporting it here since it's the most relevant repo I've found.

I hope this bug report is clear; I've tried to make it concise. If I made it too concise, let me know and I'll expand on anything that needs further explanation.

**Comment by dandv** [comment]
TL;DR - is that why [Hexchat can't connect to the irc.spotchat.org](https://github.com/linuxmint/Cinnamon/issues/5558#issue-171319454)?

**Comment by xenopeek** [comment]
For SASL you need to use port 6697, not port 6667 (see http://spotchat.org/servers/). So I think a change in mintwelcome isn't even possible as how would it know to use that port.

**Comment by xenopeek** [comment]
Many thanks for contributing to mintwelcome. Your suggestion was reviewed. For more information on our workflow and feature requests, read: https://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/faq.html
